ext/date/php_date.stub.php

```diff
@@ -412,6 +412,9 @@ public function setTimezone(DateTimeZone $timezone): DateTime {}
      */
     public function getOffset(): int {}
 
+    /** @tentative-return-type */
```

Comment:
Since these are new methods, using a tentative return type doesn't make sense, since the methods have no BC impact. Tentative return types are only useful for letting people know that they should change their already declared return type.

Reply:
What about existing code extending DateTime (like Carbon)?

Reply:
If a high profile package (like carbon) overwrote the method, then I'd say let's use tentative types for sure. Fortunately, Carbon uses magic methods, so it doesn't have one. In other cases we don't necessarily have to be considerate with internal class children when we add new methods. cc @Girgias if you have any opinion :)

Reply:
It shouldn't have a tentative return type.

If the consideration is about BC then there are way more issues, such as what if a child class implements the method with a mandatory parameter? That breaks LSP.

Reply:
@kocsismate @Girgias PR created https://github.com/php/php-src/pull/13486

Reply:
Thank you!